siemens/questa.mk

```diff
@@ -60,9 +67,21 @@ ifneq (,$(SIM_DEPS))
   endif
   ifdef	TOP_TB
     -include $(DEP_DIR)/$(TOP_TB).questa.d
+		_TOP := $(TOP_TB)
+  endif
+endif
+AC_DEPS := $(filter $(AC_TARGETS),$(MAKECMDGOALS))
+ifneq (,$(AC_DEPS))
+  # The top .d file must be called out specifically to get the ball rolling
+  # Otherwise nothing happens because there are no matches to the wildcard rule
```

Comment:
The double-comment (under `SIM_DEPS` and again under `AC_DEPS`) isn't needed, please remove this second copy of it.